core/domain/feedback_validators.py

```diff
@@ -152,6 +152,22 @@ def _get_external_id_relationships(cls, item):
             )
         return field_name_to_external_model_references
 
+    @classmethod
+    def _validate_text(cls, item):
+        """Validate that text length is less than 10000 characters.
```

Comment:
Ni: Prefer third-person for method docstring, i.e. "Validates text length is..."

Reply:
Done.